standards/8x.md

```diff
@@ -0,0 +1,111 @@
+# Specification
+
+## Summary
+A standardized interface for interacting with recurring subscriptions.
+
+##  Abstract
+This standard defines a contract interface for interacting with subscription data. The 8x suggestions come under the feelings that implementation of how a subscription is created, is left up to the implementor. If we define a standard for implementation, we'll be locking ourselves into one way of doing things.
+
+## Our Thoughts
+The idea behind a standard should be an easy way for developers to read/access data regardless of an implementation. A standard shouldn't govern the architecture of a system. It's for this reason that functions such as adding and editing seem application/protocol specific.
+
+Also features such as editing go against the principal of putting users-first. Just because we have the approval to take tokens from a user's wallet, that doesn't mean they should be uninformed of what happens. In the case of a vendor changing prices, they need to create a new plan, revoke the old one and users then need to explicitly upgrade.
+
+For variable subscriptions, that should be a completely new agreement the user enters into. They should be aware they're actually authorizing up to X per every Y. Trying to accommodate it in a standard feels too far fetched as there's multiple ways you might want to implement it.
+
+The code presented is 8x's collectible interface. The bare minimum to have recurring subscription payments for different use-cases. It's what we plan to use when the executor references a subscription contract.
+
+Rather than instantiating that specific contract, it simply instantiates a collectible interface with a particular address. We believe other developers should be able to do this for their applications too.
+
+## Motivation
+We want to set a standard for interacting with Layer 2 protocols, not how those protocols operate.
+
+#### Public View Functions
+
+###### isValidSubscription
```

Comment:
whats a reason a subscription would not be `valid`? is this the same as `active`?